content/blog/2021-03-23-obs-goes-lgpl/index.de.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "OpenBikeSensor goes LGPL"
+date: '2020-03-23'
+draft: true
+---
+
+Als Open Source Projekt freuen wir uns, bekanntgeben zu können, dass wir uns
+auf eine gemeinsame Lizenz für unsere bestehenden und zukünftigen Software-,
+Design- und Dokumentationswerke geeinigt haben. 
+
+Nach langer Diskussion hat sich die Community über die Grundsätze einer freien
+und quelloffenen Arbeit verständigt, die das Projekt hoffentlich lange
+begleiten werden.
+
+Freier Zugang zu Information sowie deren freie Verwendung ist ein Grundsatz der
+uns allen sehr wichtig ist. Daher stand auch nie wirklich zur Diskussion, eine
+richtige Open-Source-Lizenz für unsere Werke zu wählen.
```

Comment:
> richtige Open-Source-Linzenz
klingt so als gäbe es auch falsche Open-Source-Lizensen. Ich würde es entweder weglassen oder eventuell durch "freie" ersetzen.